server/src/main/java/com/example/tyfserver/admin/controller/AdminController.java

```diff
@@ -1,8 +1,39 @@
 package com.example.tyfserver.admin.controller;
 
+import com.example.tyfserver.admin.dto.AccountCancelRequest;
+import com.example.tyfserver.admin.dto.RequestingAccountResponse;
+import com.example.tyfserver.admin.service.AdminService;
+import com.example.tyfserver.member.service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
+@RequiredArgsConstructor
 public class AdminController {
 
+    private final AdminService adminService;
+
+    @PostMapping("/admin/approve/{member_id}/account")
+    public ResponseEntity<Void> approveAccount(@PathVariable("member_id") Long member_id) {
```

Comment:
```suggestion
    @PostMapping("/admin/approve/{memberId}/account")
    public ResponseEntity<Void> approveAccount(@PathVariable Long memberId) {
```
`@PathVariable`은 백에서 쓸 변수명이라 카멜케이스로 써도 돼! path이름이랑 파라미터 이름이 같으면 괄호는 빼도 돼!